### PR TITLE
Round 2 of epas edits

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/epas/14/epas_compat_cat_views/04_all_constraints.mdx [hunk 1]
```diff
@@ -15,15 +15,15 @@ The `ALL_CONSTRAINTS` view provides information about the constraints placed on
...
 | `r_owner`           | `TEXT`    | Owner of a table referenced by a referential constraint.                                                                                                                                                                                                       |
 | `r_constraint_name` | `TEXT`    | Name of the constraint definition for a referenced table.                                                                                                                                                                                                      |
-| `delete_rule`       | `TEXT`    | The delete rule for a referential constraint. Possible values are: `C` – cascade; `R` – restrict; `N` – no action                                                                                                                                              |
+| `delete_rule`       | `TEXT`    | Eelete rule for a referential constraint. Possible values are: `C` – cascade; `R` – restrict; `N` – no action                                                                                                                                              |
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/epas_compat_cat_views/04_all_constraints.mdx:24]: I think it is typo. It should be `Delete` not `Elete`

File: product_docs/docs/epas/14/epas_compat_cat_views/10_all_indexes.mdx [hunk 2]
```diff
@@ -9,25 +9,25 @@ legacyRedirectsGenerated:
...
-| `status`          | `TEXT`                  | Included for compatibility only; always set to `VALID`.                                              |
-| `partitioned`     | `CHARACTER(3)`          | Indicates that the index is partitioned. Currently, always set to `NO`.                              |
-| `temporary`       | `CHARACTER(1)`          | Indicates that an index is on a temporary table. Always set to `N`; included for compatibility only. |
-| `secondary`       | `CHARACTER(1)`          | Included for compatibility only. Always set to `N`.                                                  |
-| `join_index`      | `CHARACTER(3)`          | Included for compatibility only. Always set to `NO`.                                                 |
-| `dropped`         | `CHARACTER(3)`          | Included for compatibility only. Always set to `NO`.                                                 |
+| `logging`         | `TEXT`                  | Always set to `LOGGING`. Included only for compatibility.                                            |
```
Comment (nidhibhammar) [inline product_docs/docs/epas/14/epas_compat_cat_views/10_all_indexes.mdx:27]: Can we reverse the order of both sentences in the description column here to match with the rest of the description.

Comment (ebgitelman) [reply product_docs/docs/epas/14/epas_compat_cat_views/10_all_indexes.mdx]: Fixed

File: product_docs/docs/epas/14/epas_compat_cat_views/04_all_constraints.mdx (hunk 1 above)
Comment (ebgitelman) [reply product_docs/docs/epas/14/epas_compat_cat_views/04_all_constraints.mdx]: Fixed

**Pull request merged**